pkg/cli/cvd.go

```diff
@@ -350,20 +351,30 @@ func (c *cvdCreator) createCVDFromLocalSrcs() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
-func credentialsFactoryFromSource(source string) (CredentialsFactory, error) {
+func credentialsFactoryFromSource(source string, projectID string) (CredentialsFactory, error) {
 	switch source {
 	case NoneCredentialsSource:
-		return func() string { return "" }, nil
+		return func() client.BuildAPICredential {
+			return client.BuildAPICredential{AccessToken: "", UserProjectID: projectID}
```

Comment:
I think we don't need user project override for `NoneCredentialsSource`, so I would prefer to explicitly set `UserProjectID: ""` for this case rather than assuming that users would not provide `projectID` in their create CVD options.